### PR TITLE
D7 tests

### PR DESCRIPTION
**Pull request by greg-1-anderson**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -29,7 +29,18 @@ jobs:
...
+
+    test_drupal_7_github:
+        <<: *defaults
+        steps:
+            - checkout
+            - run:
+                name: dependencies
+                command: ./.circleci/set-up-globals.sh
+            - run:
+                name: Test D8
```
Comment (ataylorme) [inline .circleci/config.yml]: This should be Test D7 :-)

Comment (stevector) [inline .circleci/config.yml]: This line should change to `Test D7`

Comment (greg-1-anderson) [inline .circleci/config.yml]: Yes! That's confusing. 👅  Thanks.

**Comment by stevector** [comment]
+1 to merging once tests pass. It's nice to see that adding permutations is pretty straightforward (at least from the perspective of the `config.yml`)

**Pull request merged**